11_install_agent/10_install_agent_debian.txt

```diff
@@ -19,16 +19,14 @@ root@rudder-server:~# wget --quiet -O- "http://keyserver.ubuntu.com/pks/lookup?o
 
 Add Rudder project repository:
 
-* on Debian Squeeze:
+* on Debian:
 ----
 
-sudo tee /etc/apt/sources.list.d/rudder.list <<EOF
-deb http://www.rudder-project.org/apt-3.0/ $(lsb_release -cs) main
-EOF
+echo "deb http://www.rudder-project.org/apt-3.0/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
 
 ----
 
-* on Ubuntu 12.04 and following, or Debian wheezy and following:
+* on Ubuntu 12.04 and following:
```

Comment:
Since the above works also on Ubuntu, let's just stick with one command, and remove this one